charts/controller/crds/deployments.plural.sh_clusterrestoretriggers.yaml

```diff
@@ -0,0 +1,175 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.14.0
+  name: clusterrestoretriggers.deployments.plural.sh
+spec:
+  group: deployments.plural.sh
+  names:
+    kind: ClusterRestoreTrigger
```

Comment:
I actually think we could just name these `ClusterRestore` since restore is already an action (automation in pr automation is not quite descriptive of performing the automation)

Reply:
That said we could just keep the names consistent too